templates/scripts/utils.js

```diff
@@ -0,0 +1,51 @@
+const path = require("path");
+const { readdirSync, lstatSync, existsSync } = require("node:fs");
+const Mustache = require("mustache");
+
+function filterFiles(dir, fileList = [], filter = () => true) {
```

Comment:
Do we need to add unit tests to make sure core functionality/utility works fine?

Reply:
We did not add unit test for any script that work for CICD. Since We plan to run verify in CI, I think CI ensures the process works fine overall.